cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -509,6 +531,8 @@ pub enum ShiftKind {
     Left,
     RightZ,
     RightS,
+    RotateLeft,
+    RotateRight,
```

Comment:
Rotate in shift enum ... making the naming a little inconsistent from the first 3? Perhaps a more generic name (and comment update) for the Enum to show that Rotate is also housed. Also for consistency could prepend shift to Left, RightZ, and RightS, i.e. ShiftLeft, ShiftRightZ, ShiftRightS.